spec/services/invite_schools_spec.rb

```diff
@@ -504,6 +519,120 @@
+
+      InviteSchools.new.send_induction_coordinator_add_participants_email
+      expect(SchoolMailer).not_to delay_email_delivery_of(:induction_coordinator_add_participants_email)
+    end
+
+    it "sends emails to tutors who have not opted out in one of their schools" do
+      opted_out_schools = create_list(:school, 10)
+      opted_out_schools.each do |school|
+        create(:school_cohort, school: school, opt_out_of_updates: true)
+      end
+      cip_school = create(:school)
+      create(:school_cohort, school: cip_school, induction_programme_choice: "core_induction_programme")
+      induction_coordinator = create(:user, :induction_coordinator, school_ids: [cip_school.id, *opted_out_schools.map(&:id)])
+
+      InviteSchools.new.send_induction_coordinator_add_participants_email
+      expect(SchoolMailer).to delay_email_delivery_of(:induction_coordinator_add_participants_email)
+                                .with(hash_including(
+                                        recipient: induction_coordinator.email,
+                                        name: induction_coordinator.full_name,
+                                        sign_in_url: String,
+                                      ))
+    end
+
+    it "does not send emails to tutors who have participants at all of their schools" do
+      induction_coordinator = create(:user, :induction_coordinator)
+      create(:school_cohort, school: induction_coordinator.schools.first, induction_programme_choice: "core_induction_programme")
+      create(:user, :early_career_teacher, school: induction_coordinator.schools.first)
+
+      InviteSchools.new.send_induction_coordinator_add_participants_email
+      expect(SchoolMailer).not_to delay_email_delivery_of(:induction_coordinator_add_participants_email)
+    end
+
+    it "sends emails to tutors with schools without participants" do
+      schools_with_participants = create_list(:school, 10)
+      schools_with_participants.each do |school|
+        create(:school_cohort, school: school, induction_programme_choice: "core_induction_programme")
+        create(:user, :early_career_teacher, school: school)
+      end
+      school_without_participants = create(:school)
+      create(:school_cohort, school: school_without_participants, induction_programme_choice: "core_induction_programme")
+      induction_coordinator = create(:user, :induction_coordinator, school_ids: [school_without_participants.id, *schools_with_participants.map(&:id)])
+
+      InviteSchools.new.send_induction_coordinator_add_participants_email
+      expect(SchoolMailer).to delay_email_delivery_of(:induction_coordinator_add_participants_email)
+                                .with(hash_including(
+                                        recipient: induction_coordinator.email,
+                                        name: induction_coordinator.full_name,
+                                        sign_in_url: String,
+                                      ))
```

Comment:
Can we commonise these expects somehow? They are 6 lines long each, and the difference seems to be tiny (if at all)